helm/values.yaml

```diff
@@ -1258,7 +1258,7 @@ portal:
   ingress:
     enabled: true
     pathType: Prefix
-    path: /
+    path: /portal(/.*)?
```

Comment:
Coudl it be considered as a breaking change ? If yes, it should be documented (in the changelog.md ?)

Reply:
actually, if you keep / it will never work...so I cannot understand how it work like that. Maybe you know more about it ?

Reply:
I don't understand, actually if you use / it's works (eg: https://apim-4-1-x-portal.team-apim.gravitee.dev/). If you wan't to change this value, you have to add a rewrite target: nginx.ingress.kubernetes.io/rewrite-target: /$1 just like we do for /console

Reply:
if you use the values.yml like that it doesn't. For master it does because we have set a dedicated host for the portal.